docs/relations_guide.md

```diff
@@ -1,103 +1,123 @@
-# A guide to what relations to use where in the Cell Ontology
+# A guide to selecting **object properties** in the Cell Ontology (CL)
 
-## Recording location
 
-We record anatomical location by linking to terms from Uberon.
+### Recording anatomical location (general cell types)
 
-For most purposes we record the anatomical location of cells using **'part of'**. 
+To record anatomical locations for cell types, axioms are written using object properties to relate CL terms to Uberon terms.
```

Comment:
The whole document is about using object properties, AKA relations, so it sounds odd to state this specifically here.  It may be useful to have a more general introduction to state the purpose of the document and the assumptions made about prior knowledge.